actionview/test/template/date_helper_i18n_test.rb

```diff
@@ -137,6 +137,18 @@ def test_date_or_time_select_translates_prompts
     assert_equal defaults.count, @prompt_called
   end
 
+  def test_select_month_with_zero_indexed_month_names
```

Comment:
The `test_date_or_time_select_translates_prompts` test above this test seems to be out of place (it is under the `# select_month` section).  Let's ignore that for now, and put this test right below `test_select_month_given_use_short_month_option_translates_abbr_monthnames`.

Also, for anyone looking at the test in the future, a more direct name may be helpful:

```suggestion
  def test_select_month_with_frozen_month_names
```

Reply:
Fixed 👍